GW2EIEvtcParser/ParserHelpers/Images/ParserIcons.cs

```diff
@@ -40,111 +40,111 @@ internal static class ParserIcons
     public const string GenericGadgetIcon = "https://render.guildwars2.com/file/09AF0498D757B191E229C862F4AA360DA65C4FE1/1012386.png";
     // High Resolution Icons 200px
     // Ranger
-    private const string HighResGaleshot = UnknownProfessionIcon;
```

Comment:
You can rollback those changes, I'll merge the Decima file.

Reply:
Done